src/pages/Homepage/components/Hero.jsx

```diff
@@ -123,26 +130,28 @@ const [record, setRecord] = useState({
                                         <div className="flex-shrink-0 grid grid-cols-1 gap-y-6 lg:gap-y-8">
                                             <div className="w-56 h-80 rounded-lg overflow-hidden">
                                                 {record.title6 ? null : (
-                                                    <Skeleton animation="wave" variant="rectangular" style={{height: '100%'}}/>
+                                                    <Skeleton animation="wave" variant="rectangular"
+                                                              style={{height: '100%'}}/>
                                                 )}
                                                 <img
                                                     style={record.title6 ? {} : {display: 'none'}}
-                                                    src={HeroSection1Images.img6}
+                                                    src={HeroImages.img6}
                                                     onLoad={() => onLoadChange("title6")}
-                                                    alt={HeroSection1Images.title6}
+                                                    alt={HeroImages.title6}
                                                     className="w-full h-full object-center object-cover"
                                                 />
                                                 )}
                                             </div>
                                             <div className="w-56 h-80 rounded-lg overflow-hidden">
                                                 {record.title7 ? null : (
-                                                    <Skeleton animation="wave" variant="rectangular" style={{height: '100%'}}/>
+                                                    <Skeleton animation="wave" variant="rectangular"
+                                                              style={{height: '100%'}}/>
                                                 )}
                                                 <img
                                                     style={record.title7 ? {} : {display: 'none'}}
-                                                    src={HeroSection1Images.img7}
+                                                    src={HeroImages.img7}
```

Comment:
Codacy has a fix for the issue: [Replace `····················································` with `··························`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
                          src={HeroImages.img7}
```